app/controllers/people_controller.rb

```diff
@@ -2,17 +2,6 @@
 
 class PeopleController < CrudController
   include ExportController
-
-  self.permitted_attrs = %i[birthdate location
-                            marital_status updated_by name nationality nationality2 title
-                            competence_notes company_id email department_id shortname]
-
-  self.nested_models = %i[advanced_trainings activities projects
-                          educations language_skills person_roles
-                          people_skills categories]
-
-  self.permitted_relationships = %i[person_roles people_skills]
-
```

Comment:
I would not completely delete this but rather just comment it out as we will probably use this later anyways.